Dockerfile

```diff
@@ -25,12 +25,9 @@ COPY airflow.cfg ${AIRFLOW_HOME}/airflow.cfg
 COPY unittests.cfg ${AIRFLOW_HOME}/unittests.cfg
 
 # Delay creation of user and group
```

Comment:
You might want to remove this comment, since it is no longer applicable.

Reply:
oops